test/IRGen/section_structs.swift

```diff
@@ -0,0 +1,45 @@
+// RUN: %target-swift-frontend -enable-experimental-feature SymbolLinkageMarkers -parse-as-library -emit-ir %s -o - | %FileCheck %s
+
+// REQUIRES: swift_in_compiler
+
```

Comment:
Please also add generic structs, because the SIL for initializers for generic structs looks much different than for non generic structs.

Reply:
Added in section_structs_generic.swift